CMakeLists.txt

```
@@ -363,7 +363,7 @@ include(cmake/Utils.cmake)
 include(cmake/public/utils.cmake)
 
 # ---[ Version numbers for generated libraries
-set(TORCH_DEFAULT_VERSION "1.1.0")
+file(READ version.txt TORCH_DEFAULT_VERSION)
```

Can you please set it to some default value, if read failed for some reason?
```suggestion
file(READ version.txt TORCH_DEFAULT_VERSION)
if ("${TORCH_DEVALUE_VERSION} " STREQUAL " ")
  message(WARNING "Could not get version from 'version.txt'")
  set(TORCH_DEFAULT_VERSION "1.5.0")
endif()
```

---

Should be resolved by a follow up push :+1: